tests/controllers/sessions.test.js

```diff
@@ -0,0 +1,114 @@
+const tape = require('tape')
+const supertest = require('supertest')
+const server = require('../../src/server.js')
+const User = require('../../src/models/User.js')
+const { dropCollectionAndEnd } = require('../helpers/index.js')
+const { regUser, logIn } = require('../fixtures/users.json')
+const { auth } = require('../../src/constants/errors.json')
+const roles = require('../../src/constants/roles.js')
+
+tape('POST /register with new user', t => {
+  supertest(server)
+    .post('/register')
+    .send(regUser)
+    .expect(200)
+    .expect('Content-Type', /text/)
+    .end((err, res) => {
+      if (err) t.fail(err)
+      t.equal(res.text, 'registered!', 'should return \'registered!\'')
+      dropCollectionAndEnd(User, t)
+    })
+})
+
+tape('POST /register with new user and then log in', t => {
+  // register user before logging in as that user
+  supertest(server)
+    .post('/register')
```

Comment:
okay this is not a biggie, but I think unit test would be better.

Its a bit of a faff, but you could add a user fixture with a salted password in it, then unit test the login endpoint.

You want wanted to go further you could then decode the jwt..